auxx/src/Plugin.hs

```diff
@@ -43,52 +44,30 @@ import           Mode                       (AuxxMode)
 auxxPlugin ::
        (HasConfigurations, HasCompileInfo)
     => AuxxOptions
+    -> Either (WithCommandAction AuxxMode) Text
     -> (WorkerSpec AuxxMode, OutSpecs)
-auxxPlugin AuxxOptions {..} =
-    case aoAction of
-        Repl    -> worker' runCmdOuts $ runWalletRepl
-        Cmd cmd -> worker' runCmdOuts $ runWalletCmd cmd
+auxxPlugin AuxxOptions{..} = \case
+        Left WithCommandAction{..} -> worker' runCmdOuts $ \sendActions -> do
+            printAction <- getPrintAction
+            printAction "... the auxx plugin is ready"
```

Comment:
This message doesn't disrupt the REPL if the user started entering a command before the `auxxPlugin` is ready to handle commands :tada: